basics/crt/crt1.c

```diff
@@ -1,12 +1,25 @@
 extern void __wasm_call_ctors(void);
+void _Exit(int) __attribute__((noreturn));
+
+#ifdef REACTOR_RUNTIME
+extern void reactor_setup(int, char *[]);
```

Comment:
Should we make this weak?  So people don't have to define a setup function unless they want to?

Also, I'm not sure about this name. I feel like it should have wasi in the name, and also that it probably shouldn't have argv at all.    I feel like argv is a very commandliney thing and reators probably shouldn't have that at all.   But I understand this an early prototype ..

Reply:
Making it weak is a good idea, that's done.

I'm also not sure about the name. Or passing in argv. These are still things to think about.

Reply:
sgtm